src/main/java/com/salesforce/cdp/queryservice/core/QueryServiceConnection.java

```diff
@@ -38,9 +38,34 @@ public QueryServiceConnection(String url, Properties properties) {
         this.properties = properties;
         this.serviceRootUrl = url.substring(Constants.DATASOURCE_TYPE.length());
         this.properties.put(Constants.LOGIN_URL, serviceRootUrl);
+        setupDefaultClientSecretsIfRequired(serviceRootUrl, this.properties);
         this.queryExecutor = new QueryExecutor(properties);
     }
 
+    private void setupDefaultClientSecretsIfRequired(String serviceRootUrl, Properties properties) {
+        if(properties.containsKey(Constants.USER) && !properties.containsKey(Constants.USER_NAME)) {
+            properties.put(Constants.USER_NAME, properties.get(Constants.USER));
```

Comment:
Do we need new User constant? Can we stick only to userName standard?

Reply:
with DBeaver by filling in just the user name and password we should be able to connect. So the default property key for username is user.